src/components/dashboard/orderSection/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import OrderCard from '../../common/cards/orderCard';
+import OrderCard from '../../common/cards/OrderCard';
```

Comment:
This import should point to /common/OrderCard

Reply:
I actually saw these changes were needed and have them on a commit that I just pushed up and will be pulling as soon as this one is done.